### DISCUSSION TITLE
Preferred Live Chat Platforms?

### DISCUSSION THREAD

**Discussion by HumphreyYang** [QuantEcon/reading-group-2024#2] [Polls]
Hi team,

I am setting up a live chat channel for the reading group and want to know your preferred platform.

Please let me know your choice below:

**Comment by HumphreyYang** [comment]
Hi @pgrosser1 @Jiarui-ZH @mbek0605,

Please let me know your preferences : )

**Comment by HumphreyYang** [comment]
Looks like Slack wins the majority. I have created a Slack channel:
https://join.slack.com/t/quanteconomics/shared_invite/zt-2c8ohs1j9-7nXoinzzZ4FW4EjhNTQ~dQ

Please feel free to join and collaborate with the team in the channel `#reading-group-2024`. Hope this makes collaboration easier.

**Reply by mmcky** [reply]
@HumphreyYang can we use the `QuantEcon` slack account and open up a #reading-group channel?

**Reply by HumphreyYang** [reply]
Hi @mmcky,

Sure, but I think I do not have the privilege to invite people into QE slack : )

**Reply by mmcky** [reply]
Ah -- ok let me change that. I have just made you a QE workspace admin. Should be able to invite now.

**Reply by mmcky** [reply]
Also i just checked 

```
Invitations
By default, any member can invite new people to your workspace.
```

**Reply by HumphreyYang** [reply]
Cool! Thanks @mmcky. I have updated the link above.

**Reply by mmcky** [reply]
There is a channel called `#reading-group-2024`
